src/components/Table/Table/Table.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { bem } from '../../../utils';
+import styles from './Table.scss';
+
+export interface Props extends React.HTMLAttributes<HTMLTableElement> {
+    /** The content of the Table (<table />)  */
```

Comment:
the html tag `<table />` is removed from props description in Storybook.
the same happens for all the other components.

<img width="1122" alt="image" src="https://github.com/textkernel/oneui/assets/110041862/547a9569-5e1b-4e5d-8132-b418934f5b1a">

Reply:
Nice catch! Fixed